components/filtercheck/icechartcheck.tsx

```diff
@@ -0,0 +1,46 @@
+import cx from 'classnames';
+import { CheckboxProps } from 'components/forms/checkbox/component';
+import React from 'react';
+
+export interface IIceChartCheck extends CheckboxProps {
+  label: string;
+  bullet?: 'red' | 'orange' | 'yellow' | 'sky' | 'gray' | 'purple' | 'green';
+}
+
+const COLOR_MAP: any = {
+  red: 'bg-red',
+  orange: 'bg-orange',
+  yellow: 'bg-yellow',
+  sky: 'bg-sky',
+  gray: 'bg-gray',
+  none: 'bg-none',
+  purple: 'bg-purple',
+  green: 'bg-green',
+};
+
+const IceChartCheck: React.FC<IIceChartCheck> = ({ label, bullet, name, ...others }) => {
```

Comment:
Should these be checkable? I guess you already had a conversation with design. I thought they didn't have interaction

Reply:
They were checkable when I designed this but now they are not